Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -531,6 +532,13 @@ void ProgramShaderCache::CreateHeader()
 	default: SupportedESPointSize = ""; break;
 	}
 
+	switch (g_ogl_config.SupportedESTextureBuffer)
+	{
+	case 1: SupportedESTextureBuffer = "#extension GL_EXT_texture_buffer : enable"; break;
+	case 2: SupportedESTextureBuffer = "#extension GL_OES_texture_buffer : enable"; break;
+	case 3: default: SupportedESTextureBuffer = ""; break;
```

Comment:
I'd rather not have a default here, this way the compiler will warn if we add a new value to the enum and forget to update the switch.